src/main/java/org/hisp/dhis/BaseDhis2.java

```diff
@@ -799,6 +859,29 @@ protected <T> T getObjectFromUrl(URI url, Class<T> type) {
     }
   }
 
+  /**
+   * Retrieves an object using HTTP GET.
+   *
+   * @param url the fully qualified URL.
+   * @return the response.
+   * @throws Dhis2ClientException if access denied or resource not found.
+   */
+  protected String getObjectFromUrl(URI url) {
```

Comment:
I am wondering if we don't need another method to get objects. Let's reuse `getObject` and `getObjectFromUrl`. I sense there is a reason for a new method but let's see if we can reuse the existing ones.